src/vos/tests/vts_gc.c

```diff
@@ -127,7 +127,8 @@ gc_obj_update(struct gc_test_args *args, daos_handle_t coh, daos_unit_oid_t oid,
 		rc = vos_update_begin(coh, oid, epoch, &cred->tc_dkey, 1,
 				      &cred->tc_iod, &ioh, NULL);
 		if (rc != 0) {
-			print_error("Failed to prepare ZC update\n");
+			if (rc != -DER_INPROGRESS)
```

Comment:
For single vos target based test cases, unless the DTX unit tests, otherwise, the update/punch will be handled with DTX logic bypassed (via NULL @dth), so there should be no "-DER_INPROGRESS) case for GC test, right?

Reply:
Not sure.  This might be a case where I added it whilst debugging.